### PR TITLE
Allowing the user to control console output

### PR DESCRIPTION
**Pull request by SantoshDhirwani**
## Description
Added 3 verbose levels: 0, 1 and 2

0: Do not print everything, just the final step of aggregating results.
1: Print content related to SUT and benchmarking + Level 0
2: Print everything (Levels 0, 1 and 2)

## Related Issue
#92

## Motivation and Context
It is needed to allow the user to understand what is being executed at the moment. Displaying a lot of content can lead to confusion and makes it difficult to follow execution.
Also, it is possible to turn ON displaying the content, which is useful for debugging.

## How Has This Been Tested?
By executing python main.py in the bin folder.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist contributor:

- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by fayanpardo** [comment]
I still see output with flag 0
image
It seems some executions have been forgotten.
Also i have seen that GCP command results are being shown:
image
Maybe we can hide these? Maybe with a flag in gcloud commands to hide them?

**Comment by SantoshDhirwani** [comment]
I think these cannot be hidden with VERBOSE LEVELs. The levels that I've added, they are controlling the content up to some content but I can look for a flag in gcloud commands and let you know if I find something.

**Comment by SantoshDhirwani** [comment]
@boxrhcp The console output for a gcloud command can be permanently hidden with this flag `--no-user-output-enabled` .
But the question is, do we want to suppress this permanently? By using this, it won't be shown in VERBOSE_LEVEL_2 either unless the user has some debugging and google cloud skills.

**Comment by fayanpardo** [comment]
> @boxrhcp The console output for a gcloud command can be permanently hidden with this flag `--no-user-output-enabled` .
> But the question is, do we want to suppress this permanently? By using this, it won't be shown in VERBOSE_LEVEL_2 either unless the user has some debugging and google cloud skills.

The code in the deploy-sut.sh script can be modified to receive a new argument to decide if to add this flag or not in every gcloud command :) Maybe @eltalkarim has a better idea here.

**Comment by fayanpardo** [comment]
GJ! I see that you managed to hide all prompt from the rest of scripts with flag 0.
However the prints need some organization. We may need to clarify better the levels:

0: Do not print everything, just the final step of aggregating results. > right now is correctly avoiding to print the rest of scripts output but the main script is still printing things.
image

1: Print content related to SUT and benchmarking + Level 0 > right now prints everything the SUT is doing, maybe is better to move this configuration implemented to the next level. Instead, this level should be printing what is right now printing level 0 as showed in the previous image.

2: Print everything (Levels 0, 1 and 2) > is already printing everything so nothing to say here.

Make sure to add and organize prints correctly in level 1. I added several from my implementation of my algorithm but I forgot some steps for sure and it can be better :)

File: bin/main.py [hunk 1]
```diff
@@ -203,10 +225,10 @@ def find_optimal_parameters():
                 run_file(
                     ['bash', _get_path(DEPLOY_SUT_PATH), str(config['eth_param']['nodeNumber']),
                      str(interval),
-                     str(gas), '0'])
+                     str(gas), '0', '--no-user-output-enabled' if verbose_level==VERBOSE_LEVEL_0 else ''], verbose=verbose_level>=VERBOSE_LEVEL_1)
                 run_file(['python', _get_path(RUN_WORKLOAD_PATH), '--interval', str(interval),
                           '--gaslimit',
-                          str(gas)])
+                          str(gas), '--no-user-output-enabled' if verbose_level==VERBOSE_LEVEL_0 else ''], verbose=verbose_level>=VERBOSE_LEVEL_1)
```
Comment (fayanpardo) [inline bin/main.py]: Why is this here? It fails of course

**Review by fayanpardo — CHANGES_REQUESTED** [review]
Check my comments!

**Comment by SantoshDhirwani** [comment]
@boxrhcp 
0 - these are the comments added by us and make sense to print them to show execution of the tool. I would leave it the way it is.

1 - as discussed before, on slack, we agreed and I think it also makes sense to have a level that shows everything related to the SUT, so the user understands what is being benchmarked and how it is being benchmarked.

**Review by fayanpardo — APPROVED** [review]
@SantoshDhirwani now level 0 is less stressful and more uniform for the final user, level 1 indicates step by step correctly the tool execution and level 2 prints everything ;)

**Pull request merged**